ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/security/security.tsx

```diff
@@ -98,6 +98,18 @@ export class Security {
     return this.canAccess(user, Access.Share)
   }
 
+  isShared(): boolean {
+    return !(
+      this.res.accessGroups.length == 0 &&
+      this.res.accessGroupsRead.length == 0 &&
+      this.res.accessIndividuals.length == 0 &&
+      this.res.accessIndividuals.length == 0 &&
+      this.res.accessIndividualsRead.length == 0 &&
+      (this.res.accessAdministrators.length == 0 ||
+        this.res.accessAdministrators[0] === this.res.owner)
```

Comment:
Can there be more than one access administrator?

Reply:
good catch - this line should be`(this.res.accessAdministrators.length == 1 && this.res.accessAdministrators[0] === this.res.owner))

Reply:
There can be so I believe this should be changed from

```
(this.res.accessAdministrators.length == 0 || this.res.accessAdministrators[0] === this.res.owner)
```

to

```
this.res.accessAdministrators.length > 1
```

Reply:
There can be more than one access administrator so I believe I need to change this to...

```
(this.res.accessAdministrators.length == 0 ||
    (this.res.accessAdministrators[0] == this.res.owner && this.res.accessAdministrators.length == 1))
```

Reply:
It think we can remove the `.length` check too

Reply:
if a metacard is unshared then accessAdministrators should be blank or only have one entry (the owner)

Reply:
I don't think the `JSON.stringify` buys you anything here

Reply:
```
>>>> JSON.stringify(["",null, undefined, 0, 1])
<<<< "["",null,null,0,1]"
```

Reply:
I would stick with something like 
`(this.res.accessAdministrators.length == 0 || (this.res.accessAdministrators.length == 1 && this.res.accessAdministrators[0] === this.res.owner))`